transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -155,6 +155,13 @@
             .setXmlName(Attribute.DEFAULT_TIMEOUT.getLocalName())
             .setAllowExpression(true).build();
 
+    public static final SimpleAttributeDefinition MAXIMUM_TIMEOUT = new SimpleAttributeDefinitionBuilder(CommonAttributes.MAXIMUM_TIMEOUT, ModelType.INT, true)
+            .setValidator(new IntRangeValidator(300))
+            .setMeasurementUnit(MeasurementUnit.SECONDS)
+            .setDefaultValue(new ModelNode().set(31536000))
+            .setFlags(AttributeAccess.Flag.RESTART_NONE)
+            .setXmlName(Attribute.MAXIMUM_TIMEOUT.getLocalName())
```

Comment:
This is not nessesary

Reply:
I'm not sure that "setXmlName(Attribute.MAXIMUM_TIMEOUT.getLocalName())" is not necessary ?

Reply:
It defaults to the name of the attribute. Unless you want to make it different to the attribute name (which you should not be doing) then you don't need to call this.